.gitignore

```diff
@@ -1,2 +1,3 @@
 .idea
 pkg/*.gem
+Gemfile.lock
```

Comment:
then just remove it and generate a new one + add ruby 2.2.0 to .travis.yml

On Sat, Feb 21, 2015 at 9:52 AM, exAspArk <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/grosser/gettext_i18n_rails/pull/130#discussion_r25123983>
> :
>
> > @@ -1,2 +1,3 @@
> >  .idea
> >  pkg/*.gem
> > +Gemfile.lock
>
> Sure, as you wish :)
> This Gemfile.lock just doesn't work for me with ruby 2.2.0, I'll return
> it back
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/gettext_i18n_rails/pull/130/files#r25123983>.
>